style.css

```diff
@@ -1,419 +1,420 @@
+	nav ul li {
+		list-style: none;
+		padding-left: 15px;
 	}
 
-	div#svg {
-		
-		margin: 15px auto;
+	section#slideshow {
+		background-color: #EAEDEE;
 	}
 
-	div#path {
+	div#slides img {
 		display: block;
-		margin: 15px auto;
+		height: 480px;
 	}
 
-	div#mentor {
-		
-		margin: 15px auto;
-	}
 
-	
-	div#ezjobs {
-		margin: 15px auto;
+	div#onestop img {
+		padding: 30px;
+		display: block;
+		margin-left: 40%;
+		position: absolute;
 	}
-	
-	div#jobbuzz {
-		margin: 15px auto;
+
+	div#onestop h2 {
+		margin-left: 31%;
+		margin-top: 30px;
+		font-size: 2em;
+		padding: 30px;
+		letter-spacing: 0.2em;
 	}
 
-	div#photos {
-		margin: 15px auto;
+	div#svg object {
+		opacity: 0.5;
+		color: white;
+		padding: 0;
 	}
 
-	div#photos img {
-		height: 350px;
-		display: block;
-		margin: 0 auto;
+	div#svg p {
+		font-size: 0.7em;
 		text-align: center;
 	}
 
-	img.imgStyle {
-		border: 8px solid black;
-		//margin: 20px 10px 40px;
-		//position: absolute;
+	div#svg figcaption {
+		font-size: 0.8em;
+		font-style: italic;
 	}
 
-	footer {
-		width: 90%;
-		margin: 0 auto;
-	}
 
-	div#logo  {
-		margin: 15px auto;
-	}
 
-	div#logo img {
-		margin: 0 auto;
-	}
-	
-	div#contact {
-		
-		margin: 15px auto;
-		font-weight: bold;
+	object#first {
+		position: absolute;
 	}
 
-	div#social {
-		
-		margin: 15px auto;
+	object#second {
+		margin-left: 32px;
 	}
 
-	
-	form#subscribe input {
-		display: inline;
+	div#jobbuzz {
+		padding-bottom: 120px;
 	}
 
-	
-	div#social ul li {
-		display: inline;
-		margin: 10px auto;
-	}
 
-	
-	div#contact ul li {
-		display: inline;
-		margin: 10px auto;
+	section#bottom {
+		clear: both;
+		background-color: #EAEDEE;
 	}
 
-}
 
+	div#match img {
+		display: block;
+		margin-top: 30px;
+		margin-left: 43%;
+		position: absolute;
+	}
 
+	div#match h2 {
+		margin-left: 28%;
+		font-size: 2em;
+		padding: 60px;
+		letter-spacing: 0.2em;
+	}
 
-@media screen and (min-width: 769px) {
+	footer {
+		font-size: 0.8em;
+	}
 
-	header img {
-		width: 80%;
+	div#logo img {
+		margin-bottom: 10px;
 		display: block;
-		margin: 0 auto;
 	}
 
-	nav ul li {
-		display: inline;
+	div#logo p {
+		font-size: 0.8em;
+		text-align: center;
 	}
 
-	nav ul#lowerNav {
-		padding-bottom: 20px;
-		text-align: center;
+
+	div#subForm {
+		margin: 10px;
+		font-size: 0.7em;
+		background-color: #EAEDEE;
 	}
 
-	nav ul#upperNav {
-		padding-top: 20px;
-		margin-left: 61%;
+	div#para {
+		background-color: #D9D9DA;
+		padding: 1px;
 	}
 
-	
+	div#para p {
+		margin: 15px;
+		text-align: center;
+	}
 
-	div#slides img {
+	form#subscribe input {
+		margin: 15px;
+		padding: 10px;
+		border: 0px;
 		width: 70%;
-		margin: 0 auto;
 	}
 
-	div#onestop {
-		padding: 20px;
-	}
 
-	div#svg {
-		width: 60%;
-		margin: 20px auto;
+	input#submit {
+		padding: 10px;
+		background-color: #5A5A5A;
+		color: white;
 	}
 
-	div#path {
-		width: 18%;
-		float: left;
-		margin-left: 35px;
-		margin-right: 35px;
+
+	@media screen and (max-width : 768px) {
```

Comment:
when you do media query, it is better give uppper and lower bound than just upper